RubeesEat/Startup.cs

```diff
@@ -1,5 +1,7 @@
 ﻿using System.Collections.Immutable;
 using RubeesEat.Model;
+using RubeesEat.Model.DB;
+using RubeesEat.Model.InMemory;

```

Comment:
I can't remove it, because in Configure Services it still needs the InMemory Todo Repository

public void ConfigureServices(IServiceCollection services)
    {
        services.AddRazorPages();

        var todoRepository = new InMemoryTodoRepository();
        todoRepository.Add(TodoItem.Create("My first todo"));

        services.AddSingleton<ITodoRepository>(todoRepository);

        ConfigureServicesEAT(services);
    }
    
    
    If I remove the service, the Todo tests don't work.
    
    We could make a ticket to remove the Todo files, since we don't need them?

Reply:
Yes please create a ticket to remove the todo stuff and close all the opened todo issues with "won't fix"